dashboard/event_logs_types/event_logs_types.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum, unique
+
+
+@unique
+class EventLogTypes(Enum):
+    EVENT_VIEW_FILE_ACCESS = "VIEW_FILE_ACCESS"
+    EVENT_VIEW_ASSIGNMENT_PLANNING = "VIEW_ASSIGNMENT_PLANNING"
+    EVENT_VIEW_GRADE_DISTRIBUTION = "VIEW_GRADE_DISTRIBUTION"
+    EVENT_VIEW_SET_DEFAULT = "VIEW_SET_DEFAULT"
+
```

Comment:
I felt the event log should all be coming from one place instead of the `views.py` so separated as an Enum class

Reply:
some of your PR comments could be embedded inside the code as Python comments.

Reply:
Done